### PR TITLE
Added com.giadamusic.Giada

### PR DESCRIPTION
**Pull request by hfiguiere**
Giada is loop machine (music)

Includes VST support

The upstream issue to build a flatpak
https://github.com/monocasual/giada/issues/324

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.giadamusic.Giada

**Comment by hfiguiere** [comment]
bot, build com.giadamusic.Giada

**Comment by hfiguiere** [comment]
bot, build com.giadamusic.Giada

File: com.giadamusic.Giada.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 Giada Developers -->
+<component type="desktop">
```
Comment (bilelmoussaoui) [inline com.giadamusic.Giada.appdata.xml:3]: It would be nice to contribute this upstream

File: com.giadamusic.Giada.desktop [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Giada
```
Comment (bilelmoussaoui) [inline com.giadamusic.Giada.desktop:4]: Same

**Comment by bilelmoussaoui** [comment]
bot, build com.giadamusic.Giada

**Comment by hfiguiere** [comment]
Ah right. I forgot about these. Will do.

**Comment by hfiguiere** [comment]
Files submitted upstream.

**Comment by barthalion** [comment]
bot, build com.giadamusic.Giada

**Pull request closed without merging**